.circleci/config.yml

```diff
@@ -115,44 +115,44 @@ jobs:
         - run: bazel test //client_python:test_integration --test_output=streamed
 
   workbase-unit-tests:
-      docker:
-        - image: circleci/node:8.11.2
+      machine: true
       working_directory: ~/grakn
       steps:
         - checkout
-        - run: cd workbase && npm install
-        - run: cd workbase && npm test
+        - *prepare_apt_repositories
+        - *install_bazel
+        - run: bazel run @nodejs//:npm install
+        - run: bazel test //workbase:tests-unit
 
   workbase-e2e:
       machine: true
       working_directory: ~/grakn
       steps:
         - checkout
         - *prepare_apt_repositories
-        - *install_yarn_and_unzip
-        - *install_node
         - *install_bazel
+        - run: sudo apt-get -y install zip unzip
         - run: sudo apt install xvfb libxtst6 libxss1 libgtk2.0-0 -y
         - run: sudo apt install libnss3 libasound2 libgconf-2-4 -y
         - run: bazel build //:distribution --sandbox_debug
         - run: unzip bazel-genfiles/dist/grakn-core-all.zip -d bazel-genfiles/dist/
         - run: nohup bazel-genfiles/dist/grakn-core-all/grakn server start
         - run: bazel-genfiles/dist/grakn-core-all/grakn console -f `pwd`/test-end-to-end/test-fixtures/basic-genealogy.gql -k gene
-        - run: cd workbase && npm install
+        - run: bazel run @nodejs//:npm install
         - run: Xvfb :99 &
         - run: export DISPLAY=:99
-        - run: cd workbase && npm run e2e
+        - run: bazel test -s --test_output=streamed //workbase:tests-e2e
 
   build-workbase-linux:
-      docker:
-        - image: circleci/node:8.11.2
```

Comment:
I think the fail is attributed to the node version. I think we will need to revert it back to 8.11.2. What do you think @marco-scoppetta ?